isis/scripts/downloadIsisData

```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
         raise LookupError(f"{mission} is not in the list of supported missions: {supported_missions.keys()}")
 
     if mission == "legacybase":
-        args = create_rclone_arguments(dest, "legacybase_usgs:", dry_run, ntransfers)
-        rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+        args = create_rclone_arguments(dest, "legacybase_usgs:", ntransfers, kwargs)
+        rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
     elif(mission.upper() == "ALL"):
         supported_missions.pop("legacybase")
         for mission, remotes in supported_missions.items():
             for remote in remotes:
-                args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-                rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+                args = create_rclone_arguments(dest, remote, ntransfers, kwargs)
+                rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
     else:
         for remote in supported_missions[mission]:
-            args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-            rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+            args = create_rclone_arguments(dest, remote,  ntransfers, kwargs)
+            rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
```

Comment:
This swap will cause existing files to be re-downloaded? In essence, is the rclone sync vs. copy the same different from the aws s3 sync vs. copy?

Reply:
copy checks if the file exists before re-copying. It should only re-copy if 1. the same file exists on different sources and 2. the file is newer. I would say re-download if another source has a newer version of the same file is probably desired? Shouldn't happen too much after we sanitize the USGS sources.

Reply:
Agreed, we want to take newer files, but it shouldn't happen too often. The kernels are always versioned as separate files